config/crd/bases/cas-issuer.jetstack.io_googlecasclusterissuers.yaml

```diff
@@ -1,10 +1,10 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This is v1 of the Kubernetes CRD spec rather than v1 of the CAS issuer spec! We're at v1alpha1 of the CAS issuer spec I intend to add a v1beta1 when the Google CAS API publishes their GA protos

Reply:
:+1: of course